packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -165,23 +166,26 @@ Future<void> buildWindows(WindowsProject windowsProject, BuildInfo buildInfo, {
   }
 }
 
+String getCmakeWindowsArch(TargetPlatform targetPlatform) {
+  return switch (targetPlatform) {
+    TargetPlatform.windows_x64 => 'x64',
+    TargetPlatform.windows_arm64 => 'ARM64',
+    _ => throwToolExit('Unsupported target platform "$targetPlatform".'),
```

Comment:
is this user error or an internal tool bug if we reach this point? ToolExits are reserved for situations where either the user provided invalid inputs, or their environment is broken in some way we can't recover from.

Reply:
This is an internal error. Changed this to throw an `Exception` instead.